wiki/Client/Menu_tips/en.md

```diff
@@ -0,0 +1,118 @@
+- For fast combos, try toggling between left and right mouse button, or use keyboard alternative `Z`/`X`!
+- If you can't play videos, take a look at the wiki before posting a bug report!
+- Take a screenshot from within osu! by hitting `F12`.
+- Try watching others as they play using [spectator mode](/wiki/Gameplay/Spectating). Hit `F9` to toggle the extended chat interface and utilise this!
+- When a beatmap has been updated, as long as you are in online ranking mode you will be given a button to update to the latest version.
+- The wiki has a wealth of information in it. Seriously, read it.
+- Did you know you can load new beatmaps by simply double-clicking on an `osz` file or dragging it into a running osu! window?
+- You can export any replay whether it be online or offline by hitting `F2` at the [ranking screen](/wiki/Client/Interface#results-screen).
+- If you find that the 'beats' sound too early or late in every beatmap you play, try adjusting the `Universal Offset` in `Options`.
+- The fastest way to find a beatmap is to simply type part of its title/artist/creator directly into the song select screen.
+- Chat can be set to automatically close during gameplay. See the toggle buttons located at the bottom right of the screen when chat is open.
+- Adjust the volume in-game using the mouse wheel. This can be disabled in options if you wish.
+- Pause/unpause the game quickly by hitting the middle mouse button. This can be disabled in options if you wish.
+- You can gain quick access to web links by clicking the chat bubble icon in song select, just above the [score listings](/wiki/Client/Interface#rankings).
+- Customise your osu! experience with new [skins](/wiki/Skin)! Download them from the [Skinning forum](https://osu.ppy.sh/community/forums/15).
+- Hit `F8` at any time to activate the in-game chat window. You can check out the [commands availablei](/wiki/Client/Interface/Chat_console#commands-list) in chat by typing `!help` or `/help`
+- Make sure to take breaks every so often. Your hands have to serve you in life for more than just osu! [spinners](/wiki/Gameplay/Hit_object/Spinner).
+- Don't ignore [unranked](/wiki/Beatmap/Category) maps. If you find some you like and leave constructive comments, there is a much higher chance of them being [ranked](/wiki/Beatmap_ranking_procedure)!
+- You can play songs together with other players in [multiplayer mode](/wiki/Client/Interface/Multiplayer)!
+- Ever wondered where someone was from? Hit `F9` and hover their user panel!
+- Hit `F5` at the song select screen to refresh and load any new songs from disk.
+- Hit `Shift` + `F12` anywhere in-game to take a screenshot and automatically upload it to the internet.
+- You can customise the keys you use to play osu! under `Options` > `Input`.
+- You can search for [specific conditions](/wiki/Client/Interface#search) at song select, like "ar>8 od=9". Supported keywords are: `ar`, `od`, `hp`, `cs`, `bpm`, `length` and `drain` (time in seconds).
```

Comment:
there are way more filters supported according to https://osu.ppy.sh/wiki/en/Client/Interface#search